tpl/reflect/init.go

```diff
@@ -11,6 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package reflect provides template functions for run-time object reflection.
```

Comment:
We need to do better than that; what functionality does the package provide?

Reply:
`Package reflect provides template functions for identifying object types.`?

Offer a better description.